### PR TITLE
Routeros - Fix displaying distance in charts

### PR DESCRIPTION
**Pull request by Martin22**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 14300`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: LibreNMS/OS/Routeros.php [hunk 1]
```diff
@@ -378,7 +378,7 @@ public function discoverWirelessDistance()
                 '60G: Sta > ' . $entry['mtxrWl60GStaRemote'],
                 $entry['mtxrWl60GStaDistance'],
                 1,
-                1000
+                100000
```
Comment (murrant) [inline LibreNMS/OS/Routeros.php:381]: Interesting, considering the MIB says this is defined in meters not centimeters :D

File: LibreNMS/OS/Routeros.php [hunk 2]
```diff
@@ -378,7 +378,7 @@ public function discoverWirelessDistance()
                 '60G: Sta > ' . $entry['mtxrWl60GStaRemote'],
                 $entry['mtxrWl60GStaDistance'],
```
Comment (murrant) [inline LibreNMS/OS/Routeros.php:379]: This should be divided by the divisor I think.

File: LibreNMS/OS/Routeros.php (hunk 1 above)
Comment (Martin22) [reply LibreNMS/OS/Routeros.php:381]: The graph shows kilometres and not metres, so the MIB is saying it wrong. The connection is not really 18 km.

Before changes:
image

After changes:
image

Comment (murrant) [reply LibreNMS/OS/Routeros.php:381]: Yes, the graph data is scaled to km for consistency.

Speaking of consistency, do all devices/firmwares report this value the same?

Comment (Martin22) [reply LibreNMS/OS/Routeros.php:381]: > Yes, the graph data is scaled to km for consistency.

I understand, that's not a problem.

> 
> Speaking of consistency, do all devices/firmwares report this value the same?

Tested on the first firmware with 60G support, the middle and the latest, the same value is returned everywhere. Also tested Cube 60G, WAP 60G, RBLHG 60G (no longer sold) and the same values are returned everywhere.

Comment (murrant) [reply LibreNMS/OS/Routeros.php:381]: Thanks for confirming.   Mikrotik isn't the best at snmp (but also not the worst)

**Comment by Martin22** [comment]
@murrant 
Why was this PR not included in the new version? I don't understand that some tests don't pass, but it's a bit strange that others where there is only a different version of php pass, I see it more of a bug elsewhere than in the PR in question.

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**